testData/preferences/demoUserPrefs/20558561-dc5b-4973-85f3-9bca975090cb.json5

```diff
@@ -0,0 +1,2 @@
+// Empty user key
```

Comment:
@javihernandez I've updated the 20 empty demo user preferences as you described.

Reply:
@javihernandez, since you've manually loaded a number of empty preferences safes as well as prefs safes for Roy and Quint into the production and staging databases for the August demo, do you think the step that loads 20 empty demo user preferences and prefs for Roy and Quint can be removed from this pull request? Thanks.

Reply:
@cindyli, updating the views in the data is simillar in that it's only done once.  We need to update the views for this new dataloader, but thereafter, re-loading the views every time the snapsets are updated is not necessary.  Do you think the step that loads the latest views can be removed, once the views are  installed?  Or, is there a chance that the views will require regular updates in the future?

Reply:
@klown, views could be updated with any code change. I think it's worthwhile to have views updated via the data loader.

Reply:
@cindyli Okay, then I can see two ways to do it, but I don't know enough about databases to determine which is the better approach:
1. Delete the current views from the database and load in the latest views.
1. Get the current views form the database, determine the version number, and then preform an update using the latest views data.

The first is straight forward but might be expensive (?).  What do you think?

Reply:
Either way is fine. I agree the first approach is more straight forward. Does "expensive" you mentioned refer to the downtime when views are not available in between after-the-deletion and before-the-reload?

Reply:
"Expensive" means either a  lot of time, or a lot of memory or both.  Time is the more important of the two here.  I assume the chances of running out memory (the worst case scenario) is unlikely.

Reply:
I'm not sure how much downtime would the deletion cause. We could either do a test or go straight with the second approach. Thanks to [Multi-Version Concurrency Control](http://docs.couchdb.org/en/stable/intro/consistency.html#no-locking) that CouchDB uses to manage concurrent access to the database, there won't be any downtime at updating documents.